pennylane/io.py

```diff
@@ -433,23 +454,6 @@ def from_qasm(quantum_circuit: str, measurements=False):
     >>> circuit()
     [tensor(1., requires_grad=True)]
 
-    .. note::
-
-        The ``measurements`` keyword allows one to add a list of PennyLane measurements
-        that will **override** any terminal measurements present in the QASM code,
-        so that they are not performed before the operations specified in ``measurements``.
-
-    By default, ``from_qasm`` will remove any measurements that are present in the QASM code.
-    If the QASM code contains measurements, set ``measurements=None`` to keep them in the
-    output of ``from_qasm``.
-
-    .. warning::
-
-        The current default behaviour of removing measurements in the QASM code is deprecated
-        and will be changed in a future release. Starting in version ``0.38``, ``from_qasm``
-        will keep the measurements from the QASM code by default. To remove all measurements,
-        set ``measurements=[]`` which overrides the existing measurements with an empty list.
-
```

Comment:
Discussed offline, concluded that the note is redundant because of the information in the Args section, but we will keep the information explaining the default behaviour when to `measurements` are passed.